### PR TITLE
[DOC release] Add documentation for click 2nd argument

### LINKED ISSUE #13397: Should the click helper's 2nd argument be documented?
Looks like the `click` helper

http://emberjs.com/api/classes/Ember.Test.html#method_click
https://guides.emberjs.com/v2.5.0/testing/acceptance/#toc_asynchronous-helpers

Accepts `context` as a second arg.

The folks over on the guides repo are curious if the folks over here believe that it should be documented.

https://github.com/emberjs/guides/issues/1386

### PR DESCRIPTION
**Pull request by ladyleet**
Fixes #13397

### CODE REVIEW

**Comment by ladyleet** [comment]
This is my first PR. Thanks @toddjordan for walking me through it. 👍

File: packages/ember-testing/lib/helpers.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ asyncHelper('visit', visit);
 
   @method click
   @param {String} selector jQuery selector for finding element on the DOM
+  @param {Object} context A DOM Element, Document, or jQuery to use as context
```
Comment (ladyleet) [inline packages/ember-testing/lib/helpers.js]: is object okay for this?

**Comment by ladyleet** [comment]
This is in response to #13397 @pixelhandler @sethbrasile

File: packages/ember-testing/lib/helpers.js (hunk 1 above)
Comment (pixelhandler) [inline packages/ember-testing/lib/helpers.js]: @ladyleet yeah given the choices, which you quoted <http://api.jquery.com/jQuery/#jQuery1>, in the description `Object` seems fine,

Comment (jayphelps) [inline packages/ember-testing/lib/helpers.js]: Isn't it `Element|jQuery` ?

**Comment by ladyleet** [comment]
@pixelhandler is there anything else I need to do on this PR?

File: packages/ember-testing/lib/helpers.js (hunk 1 above)
Comment (pixelhandler) [inline packages/ember-testing/lib/helpers.js]: @jayphelps `Element|Document|jQuery` ?

Comment (ladyleet) [inline packages/ember-testing/lib/helpers.js]: @pixelhandler @jayphelps would using `Element | Document | jQuery` to cause issues with google closure compiler? (per @blesh #sobenlesh)

Comment (jayphelps) [inline packages/ember-testing/lib/helpers.js]: @pixelhandler indeed, seems jQuery docs is missing that in the types, but claims it in the description.

@ladyleet I haven't the slightest idea.

**Comment by pixelhandler** [comment]
@ladyleet perhaps @emberjs/core can merge

File: packages/ember-testing/lib/helpers.js (hunk 1 above)
Comment (pixelhandler) [inline packages/ember-testing/lib/helpers.js]: I actually think {Object} as the description expailns the type of object

Comment (jayphelps) [inline packages/ember-testing/lib/helpers.js]: I don't _personally_ care too much but seemingly using `{Object}` is basically just saying "I dunno" or "any" so not really helpful.

**Comment by locks** [comment]
😁  congrats on your first PR!

File: packages/ember-testing/lib/helpers.js (hunk 1 above)
Comment (jayphelps) [inline packages/ember-testing/lib/helpers.js]: @ladyleet see the above convo ^^^ ?

Welcome to github, this is called bikeshedding.

**Pull request merged**